core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -80,12 +80,16 @@ public static void main(String[] args) {
       LOG.info(USAGE);
       System.exit(-1);
     }
+    Mode mode = null;
     try {
-      format(Mode.valueOf(args[0].toUpperCase()));
+      mode = Mode.valueOf(args[0].toUpperCase());
     } catch (IllegalArgumentException e) {
       LOG.error("Unrecognized format mode: {}", args[0]);
       LOG.error("Usage: {}", USAGE);
       System.exit(-1);
+    }
+    try {
+      format(mode);
```

Comment:
If user input is invalid it's good to print usage info, but if something internal breaks we should not. The raw error would say something like `No enum constant alluxio.cli.Format.MASTE`. If we don't care about friendliness of running this class (it's almost always invoked indirectly by `bin/alluxio`), I think your suggestion makes sense. I'd lean towards friendliness, but don't feel strongly either way. Which do you prefer?